HW1/P4/P4.py

```diff
@@ -0,0 +1,50 @@
+import findspark
+findspark.init()
+import pyspark
+import re
+import csv
+
+from P4_bfs import *
+
+sc = pyspark.SparkContext(appName="P4")
+sc.setLogLevel('ERROR')
+
+#load in the list of characters and appearances in comic books
+with open("source.csv", "rb") as inputfile:
+	reader = csv.reader(inputfile, quotechar='"')
```

Comment:
I think you can do most of your processing in spark instead of in the python driver